lib/business_readiness/loader.rb

```diff
@@ -30,142 +36,23 @@ def tags_for_row(row)
     end
 
     def create_all_tags
-      {
-        "sector_business_area" => all_sector_business_area,
-        "employ_eu_citizens" => all_employ_eu_citizens,
-        "doing_business_in_the_eu" => all_doing_business_in_the_eu,
-        "regulations_and_standards" => all_regulations_and_standards,
-        "personal_data" => all_personal_data,
-        "intellectual_property" => all_intellectual_property,
-        "receiving_eu_funding" => all_receiving_eu_funding,
-        "public_sector_procurement" => all_public_sector_procurement
-      }
+      tags = {}
+      facets.each do |facet|
```

Comment:
Here you could do `facets.each_with_object({}) do |facet, tags|` to avoid the variable definition and return below.